tsconfig.json

```diff
@@ -17,7 +17,11 @@
     "incremental": true,
     "noUncheckedIndexedAccess": true,
     "noImplicitAny": true,
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "baseUrl": ".",
```

Comment:
does it work without `baseUrl`?

Reply:
Even if it does, it's not supposed to [see here](https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases) where it says "Each of the "paths" are relative to the baseUrl location"